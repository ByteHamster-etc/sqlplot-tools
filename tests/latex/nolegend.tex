line1
% IMPORT-DATA test test.data
line2
%% MULTIPLOT(funcname|nolegend) SELECT LOG(testsize) / LOG(2) AS x, bandwidth AS y, MULTIPLOT
%% FROM test WHERE host='earth' ORDER BY MULTIPLOT,x
this is the end
