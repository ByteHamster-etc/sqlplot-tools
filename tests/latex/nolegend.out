line1
% IMPORT-DATA test test.data
line2
%% MULTIPLOT(funcname|nolegend) SELECT LOG(testsize) / LOG(2) AS x, bandwidth AS y, MULTIPLOT
%% FROM test WHERE host='earth' ORDER BY MULTIPLOT,x
\addplot coordinates { (10.1699,2.12566e+10) (11.0875,2.12068e+10) (11.6439,2.12568e+10) (12.0444,2.12565e+10) (12.6147,2.12549e+10) (13.0224,2.10335e+10) (13.5999,2.11052e+10) (14.0112,2.11423e+10) (14.3309,2.11649e+10) (14.5925,2.1178e+10) (14.8138,2.11881e+10) (15.0056,2.01671e+10) (15.3264,1.54346e+10) (15.5887,1.54457e+10) (16.0028,1.54706e+10) (16.5868,1.54763e+10) (17.0014,1.54782e+10) (17.5859,1.54527e+10) (18.0007,1.54039e+10) (18.5854,1.54242e+10) (19.0004,1.5424e+10) (19.5852,1.54265e+10) (20.0002,1.54166e+10) (20.3221,1.52702e+10) (20.5851,1.52696e+10) (20.8075,1.52691e+10) (21.0001,1.43805e+10) (21.17,1.43101e+10) (21.322,1.36672e+10) (21.4595,1.18576e+10) (21.585,1.09755e+10) (22,7.46265e+09) (22.322,5.23999e+09) (22.585,4.38139e+09) (22.8074,4.03316e+09) (23,3.84756e+09) (23.1699,3.74337e+09) (23.3219,3.66671e+09) (23.585,3.58156e+09) (23.8074,3.54338e+09) (24,3.52208e+09) (24.3219,3.50742e+09) (24.585,3.50313e+09) (24.8074,3.50257e+09) (25,3.50191e+09) (26,3.50248e+09) (27,3.50222e+09) (28,3.50208e+09) (29,3.50212e+09) (30,3.50154e+09) (31,3.50165e+09) (32,3.50154e+09) (33,3.50111e+09) (34,3.50009e+09) };
% \addlegendentry{funcname=ScanRead64PtrUnrollLoop};
\addplot coordinates { (10.1699,2.00049e+10) (11.0875,2.00592e+10) (11.6439,2.01016e+10) (12.0444,2.00787e+10) (12.6147,2.01033e+10) (13.0224,1.98443e+10) (13.5999,1.99341e+10) (14.0112,1.99704e+10) (14.3309,1.99977e+10) (14.5925,2.00171e+10) (14.8138,2.00244e+10) (15.0056,1.95887e+10) (15.3264,1.33525e+10) (15.5887,1.33527e+10) (16.0028,1.33532e+10) (16.5868,1.33536e+10) (17.0014,1.33535e+10) (17.5859,1.33537e+10) (18.0007,1.33538e+10) (18.5854,1.33544e+10) (19.0004,1.33554e+10) (19.5852,1.33571e+10) (20.0002,1.33223e+10) (20.3221,1.25999e+10) (20.5851,1.25974e+10) (20.8075,1.25935e+10) (21.0001,1.18143e+10) (21.17,1.17322e+10) (21.322,1.11514e+10) (21.4595,9.63556e+09) (21.585,8.83049e+09) (22,5.70303e+09) (22.322,3.76486e+09) (22.585,2.97082e+09) (22.8074,2.62547e+09) (23,2.44873e+09) (23.1699,2.35262e+09) (23.3219,2.28865e+09) (23.585,2.22295e+09) (23.8074,2.19541e+09) (24,2.1806e+09) (24.3219,2.1704e+09) (24.585,2.16835e+09) (24.8074,2.16741e+09) (25,2.16724e+09) (26,2.16707e+09) (27,2.1674e+09) (28,2.16717e+09) (29,2.16721e+09) (30,2.16729e+09) (31,2.16717e+09) (32,2.16733e+09) (33,2.16723e+09) (34,2.1669e+09) };
% \addlegendentry{funcname=ScanWrite64PtrUnrollLoop};
this is the end
